tests/desktop/consumer_pages/test_users_account.py

```diff
@@ -35,7 +38,8 @@ def test_user_can_sign_in_and_sign_out_from_home_page(self, mozwebqa):
         home_page.go_to_homepage()
 
         home_page.header.click_sign_in()
-        home_page.login(user="default")
+        acct = home_page.get_default_user()
```

Comment:
Is there a reason that the tests in this file are using the default user instead of creating a new user? It seems to me that it would be safer to always create a new user, unless there is a specific reason for needing an existing user for a test.